packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.pcss

```diff
@@ -0,0 +1,16 @@
+@import '../../../global/global.pcss';
+@import '../facet-common.pcss';
+@import '../facet-search/facet-search.pcss';
+
+.box-container {
+  @apply flex items-stretch inline-block;
+}
+
+.value-box .value-label,
+.value-box-count {
+  @apply w-full;
+}
+
+.value-box.selected {
+  box-shadow: 0 0 0 1px var(--atomic-primary);
```

Comment:
![Screen Shot 2022-05-27 at 9 20 11 AM](https://user-images.githubusercontent.com/4923043/170707330-606c3131-5a29-46c3-a916-d8a0509d43e5.png)
the selected would have a blue (primary) `border` & `color` for the text in the mockups

Reply:
The other box value kind of had a css "hack" to prevent the facet from changing size when selected. It should be fin to use the border for this one 🤞